reconcile/prometheus_rules_tester.py

```diff
@@ -17,6 +17,10 @@
 from reconcile.status import ExitCodes
 from reconcile.utils.structs import CommandExecutionResult
 
+# This comes from prometheus-operator. It is the largest configmap that they will
+# create. It is also the largest PrometheusRule yaml file they will process to add it
+# to the configmap.
+MAX_CONFIGMAP_SIZE = 1 * 1024 * 1024 / 2
```

Comment:
Is there any reason to multiply it by 1? 
```suggestion
MAX_CONFIGMAP_SIZE = (1024 * 1024) / 2
```

Reply:
Not many, it came from the combination kubernetes and prometheus operator code :-D

* https://github.com/kubernetes/kubernetes/blob/976a940f4a4e84fe814583848f97b9aafcdb083f/staging/src/k8s.io/api/core/v1/types.go#L6112
* https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/rules.go#L45

I guess people like to separate unit conversions from values which makes kind of sense. In that spirit I have rewritten it as `0.5 * 1024 * 1024`  in a0a7f9bc